code/modules/research/techweb/all_nodes.dm

```diff
@@ -274,10 +274,21 @@
 	display_name = "Applied Bluespace Research"
 	description = "Using bluespace to make things faster and better."
 	prereq_ids = list("bluespace_basic", "engineering")
-	design_ids = list("bs_rped","minerbag_holding", "bluespacebeaker", "bluespacesyringe", "bluespacebodybag", "phasic_scanning", "roastingstick", "ore_silo", "antivirus3")
+	design_ids = list("bs_rped","minerbag_holding", "bluespacebeaker", "bluespacesyringe", "bluespacebodybag", "phasic_scanning", "roastingstick", "antivirus3")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	export_price = 5000
 
+/datum/techweb_node/material_distribution
+	id = "material_distribution"
+	tech_tier = 5
+	display_name = "Bluespace Material Distribution"
+	description = "Using bluespace to replace the Quartermaster."
+	prereq_ids = list("practical_bluespace", "bluespace_travel")
+	design_ids = list("ore_silo")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 8000)
+	export_price = 10000
+	hidden = TRUE
```

Comment:
This could be a config if you want to be nice to downstreams

Reply:
Could. I have no interest in doing so however.